packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -464,6 +473,8 @@ export class Function extends FunctionBase {
 
   private _logGroup?: logs.ILogGroup;
 
+  private readonly profilingGroup?: ProfilingGroup;
```

Comment:
Doesn't look like this instance variable is used beyond the constructor. Local variables should be sufficient here.